src/NuGet.Clients/VsExtension/source.extension.vsixmanifest

```diff
@@ -8,12 +8,12 @@
     <PreviewImage>Resources/nuget_256.png</PreviewImage>
   </Metadata>
   <Installation InstalledByMsi="false" AllUsers="true">
-    <InstallationTarget Id="Microsoft.VisualStudio.Pro" Version="[14.0,]" />
-    <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[14.0,]" />
-    <InstallationTarget Id="Microsoft.VisualStudio.VWDExpress" Version="[14.0,]" />
-    <InstallationTarget Id="Microsoft.VisualStudio.VPDExpress" Version="[14.0,]" />
-    <InstallationTarget Id="Microsoft.VisualStudio.VSWinExpress" Version="[14.0,]" />
-    <InstallationTarget Id="Microsoft.VisualStudio.VSWinDesktopExpress" Version="[14.0,]" />
+    <InstallationTarget Id="Microsoft.VisualStudio.Pro" Version="14.0" />
```

Comment:
I don't know how the VSIX installer interprets "14.0", as to whether that implicitly allows higher major or minor versions. IMO it may be safer to specify it as `[14.0,15.0)` so that you apply to all 14.x versions.